test/mongoShell.test.ts

```diff
@@ -28,13 +29,18 @@ if (!isWindows) {
     testsSupported = false;
 }
 
-suite("MongoShell", function (this: Mocha.Suite): void {
-    function testIfSupported(title: string, fn?: Mocha.Func | Mocha.AsyncFunc): void {
-        if (testsSupported) {
-            test(title, fn);
-        } else {
-            test(title);
-        }
+suite("MongoShell", async function (this: Mocha.Suite): Promise<void> {
+    // https://aka.ms/AA8o5r3
```

Comment:
Lol did you create an aka.ms link just for this? If so, pretty overkill for a comment in test code...

Reply:
Lol my b, I got into the habit of doing that 😅